service/controller/resource/scrapeconfigs/test/aws/case-0-cluster-api.golden

```diff
@@ -437,6 +437,14 @@
   # Add installation label.
   - target_label: installation
     replacement: test-installation
+  # Add role label.
+  - source_labels: [__meta_kubernetes_node_label_role]
+    target_label: role
+  # If role is empty, we default to worker
+  - source_labels: [__meta_kubernetes_node_label_role]
+    regex: null
+    target_label: role
+    replacement: worker
```

Comment:
Is this related to adding Celestial rules? Sorry, I didn't notice that before.

Reply:
Yes it is as this alert needs the role label on the kubelet:
![image](https://user-images.githubusercontent.com/2787548/107500830-ee2b1780-6b96-11eb-8e63-2b9ee1a098bb.png)

Reply:
I am sorry, I should have created another PR

Reply:
Ah OK, if you could just mention adding the label in the CHANGELOG it'd be cool. Sorry to be a pain, it's just it's good to have changes accounted for in the CHANGELOG so you can tell where and why they appeared (which BTW is super useful when you're trying to remove things down the line, cause you can tell what maybe affected by a removal).

Reply:
You are definitely right here :D That's why I asked you for another review :D